components/shared/AppSidebar/AppSidebar.tsx

```diff
@@ -41,18 +41,15 @@ const InsightsList = ({
   );
 };
 
+export const WORKSPACE_UPDATED_EVENT = "workspaceUpdated";
```

Comment:
I opted for the constant in this file as opposed to a seprate file. Maybe if we have more custom events in the future, it'd make sense to pull this out.